management/src/clusterm/manager/client_test.go

```diff
@@ -89,53 +100,59 @@ func (s *managerSuite) TestPostSuccess(c *C) {
 		url: baseURL,
 	}
 
+	var reqHostGroupBody bytes.Buffer
+	c.Assert(json.NewEncoder(&reqHostGroupBody).Encode(testReq1HostGroupBody), IsNil)
+
+	var testFlags ActionFlags
 	tests := map[string]struct {
 		expURLStr string
 		nodeName  string
-		extraVars string
+		flags     ActionFlags
 		exptdBody []byte
-		cb        func(name, extraVars string) error
+		cb        func(name string, flags ActionFlags) error
 	}{
 		"commission": {
 			expURLStr: fmt.Sprintf("http://%s/%s/%s", baseURL, PostNodeCommissionPrefix, testNodeName),
 			nodeName:  testNodeName,
-			extraVars: "",
-			exptdBody: []byte{},
+			flags:     ActionFlags{"", ansibleMasterGroupName},
+			exptdBody: reqHostGroupBody.Bytes(),
```

Comment:
ok, will add another okReturner which will unMarshal the `r.Body` and do an explicit `c.Assert(bosy.HostGroup, Equals, expHostGroup)`